sandbox/tests/test scenes/archive assemblies/01 - archive assemblies.appleseed

```diff
@@ -0,0 +1,130 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- File generated by appleseed.so version 1.6.0-beta (SSE 4.2). -->
+<project format_revision="13">
+    <search_paths>
+        <search_path>
+            ../_assets/textures
+        </search_path>
+        <search_path>
+            ../bsdf/disneybrdf
```

Comment:
I don't think it's a good idea for test scenes to reference each others. What are you using from that directory?